demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -268,7 +268,6 @@ def validate_pack_meta_file(self):
             self._is_approved_usecases(),
             self._is_approved_tags(),
             self._is_price_changed(),
-            self._is_approved_tags(),
```

Comment:
Did we intentionally delete this?

Reply:
Yes, It runs twice-
self._is_approved_usecases(),
**self._is_approved_tags(),**
self._is_price_changed(),
**self._is_approved_tags()**